Samples/UnifiedAPI/ArmoniK.Samples.Client/Program.cs

```diff
@@ -151,8 +154,11 @@ private static async Task Main(string[] args)
       rootCommand.SetHandler(() =>
                              {
                                logger_.LogInformation("Running Simple execution test with UnifiedApi");
-                               new SimpleUnifiedAPI(configuration_,
-                                                    factory).SimpleExecution();
+                               var submitter = new SimpleUnifiedAPI(configuration_,
```

Comment:
I think the recommended way to call the dispose method is

`using new SimpleUnifiedAPI(configuration_,
                                                    factory).SimpleExecution();`